keyboard-racing/src/components/Home/Home.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+class Home extends React.Component {
+
+  componentDidMount(){
+    const urlParams = new URLSearchParams(window.location.search);
+    const myParam = urlParams.get('code');
+    if(myParam){
```

Comment:
Tsk tsk.. add space

Reply:
I thought about it. You mean between const and if statement ?

Reply:
no I mean `if(myParam){ ==> if (myParam) {` LOL i was just being a meanie

Reply:
OHHHH lol I made myself a liar.